src/app/api/hopeFuelList/items/route.js

```diff
@@ -0,0 +1,97 @@
+import { NextResponse } from "next/server";
+
+import db from "../../../utilites/db";
+
+async function retrieveCurrentMonthHopeFuelCards(page, limit) {
+    const offset = (page - 1) * limit;
+    console.log("Page:", page, "Limit:", limit, "Offset:", offset);
+
+  let query = `
+   SELECT
+            t.HopeFuelID,
+            c.Name,
+            c.Email,
+            c.CardID,
+            t.TransactionDate,
+            t.Amount,
+            curr.CurrencyCode,
+            t.Month,
+            GROUP_CONCAT(DISTINCT ss.ScreenShotLink SEPARATOR ', ') AS ScreenShot,
+            c.ManyChatId ,
+            GROUP_CONCAT(DISTINCT a.AwsId SEPARATOR ', ') AS 'FormFilledPerson',
+            ts.TransactionStatus,
+            n.Note AS Note
+        FROM Transactions t
+        LEFT JOIN Customer c ON t.CustomerID = c.CustomerId
+        LEFT JOIN Wallet w ON t.WalletID = w.WalletId
+        LEFT JOIN Currency curr ON w.CurrencyId = curr.CurrencyId
+        LEFT JOIN Note n ON t.NoteID = n.NoteID
+        LEFT JOIN ScreenShot ss ON t.TransactionID = ss.TransactionID
+        LEFT JOIN TransactionAgent ta ON t.TransactionID = ta.TransactionID
+        LEFT JOIN Agent a ON ta.AgentID = a.AgentId
+        LEFT JOIN (
+            SELECT 
+                fs.TransactionID,
+                fs.TransactionStatusID
+            FROM FormStatus fs
+            INNER JOIN (
+                SELECT 
+                    TransactionID,
+                    MAX(FormStatusID) AS LatestFormStatusID
+                FROM FormStatus
+                GROUP BY TransactionID
+            ) latest_fs ON fs.FormStatusID = latest_fs.LatestFormStatusID
+        ) current_fs ON t.TransactionID = current_fs.TransactionID
+        LEFT JOIN TransactionStatus ts ON current_fs.TransactionStatusID = ts.TransactionStatusID
+        WHERE 
+            t.TransactionDate >= DATE_FORMAT(NOW(), '%Y-%m-01') 
+            AND t.TransactionDate < DATE_ADD(DATE_FORMAT(NOW(), '%Y-%m-01'), INTERVAL 1 MONTH)
+        GROUP BY 
+    t.HopeFuelID, 
+    c.Name, 
+    c.Email, 
+    c.CardID, 
+    t.TransactionDate, 
+    t.Amount, 
+    curr.CurrencyCode, 
+    t.Month, 
+    c.ManyChatId, 
+    ts.TransactionStatus, 
+    n.Note
+    LIMIT ${limit} OFFSET ${offset};
```

Comment:
```suggestion
    LIMIT ? OFFSET ?;
`;

const result = await db.execute(query, [limit, offset]);
```

This can introduce the sql injection attack. Make sure that all of the data are sanitized before doing anything